letsencrypt/rootfs/etc/cont-init.d/file-structure.sh

```diff
@@ -46,7 +46,12 @@ echo -e "dns_cloudxns_api_key = $(bashio::config 'dns.cloudxns_api_key')\n" \
       "dns_inwx_url = https://api.domrobot.com/xmlrpc/\n" \
       "dns_inwx_username = $(bashio::config 'dns.inwx_username')\n" \
       "dns_inwx_password = $(bashio::config 'dns.inwx_password')\n" \
-      "dns_inwx_shared_secret = $(bashio::config 'dns.inwx_shared_secret')" > /data/dnsapikey
+      "dns_inwx_shared_secret = $(bashio::config 'dns.inwx_shared_secret')\n" \
+      "dns_google_domains_access_token = $(bashio::config 'dns.google_domains_access_token')\n" > /data/dnsapikey
+
+if bashio::config.exists 'dns.google_domains_zone'; then
+      echo -e "dns_google_domains_zone = $(bashio::config 'dns.google_domains_zone')\n" >> /data/dnsapikey
+fi
```

Comment:
The `dns.google_domains_zone` config is optional just like many others, why do we need to special case it here? Can we not just assign it no matter whether the config is set or not?

Reply:
I think the plugin does not treat unset the same as set to an empty string.

Reply:
Hm, can you double check if that really is the case? Seems weird to me that all the other plug-ins don't need that special casing...